libs/core/langchain_core/output_parsers/json.py

```diff
@@ -123,7 +123,9 @@ def _type(self) -> str:
         return "simple_json_output_parser"
 
 
-# For backwards compatibility
-SimpleJsonOutputParser = JsonOutputParser
-parse_partial_json = parse_partial_json
-parse_and_check_json_markdown = parse_and_check_json_markdown
+__all__ = [
```

Comment:
@cbornet i think __all__ or an explicit redclare of the import is better (`from blah import foo as foo`)

Reply:
Sure. I think SimpleJsonOutputParser should stay for backward compatibility ?